json-rpc/eth_call.md

```diff
@@ -0,0 +1,113 @@
+
+
+# `eth_call`
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC-2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+Specification | Description 
+--|--
+1         | `eth_call` MUST NOT affect on-chain state|
+2         | `eth_call` MUST NOT accept requests that are not in valid JSON format
```

Comment:
I think this is implicit, since the RPC is JSON-RPC so it is unnecessary to repeat it here.  Similar to how we don't specify how TCP or HTTP works in this specification as it is handled at the transport layer.